sdk-base/src/main/java/com/mapbox/maps/plugin/locationcomponent/LocationConsumer.kt

```diff
@@ -37,4 +46,10 @@ interface LocationConsumer {
    * This will apply to all upcoming updates.
    */
   fun onPuckBearingAnimatorDefaultOptionsUpdated(options: ValueAnimator.() -> Unit)
+
+  /**
+   * Update [ValueAnimator] options that will be used to animate between accuracy radius [Double] updates by default.
+   * This will apply to all upcoming updates.
+   */
+  fun onPuckAccuracyRadiusAnimatorDefaultOptionsUpdated(options: ValueAnimator.() -> Unit)
```

Comment:
Same here, should we consider adding methods to this interface breaking change? If user implement the LocationConsumer themselves, this could break their code.